### PR TITLE
- The Serializer is not working due to missing Symfony\Component\Seri…

### PR DESCRIPTION
**Pull request by ThomasTr**
Pull Request for Issue #1

### CODE REVIEW

File: config/packages/framework.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ framework:
                 confirm:
                     from: open
                     to: confirmed
-                approve:
+                approved:
```
Comment (chr-hertel) [inline config/packages/framework.yaml]: the transition should still be `approve` and the state is `approved`

Comment (ThomasTr) [reply config/packages/framework.yaml]: ...then you will get the error "Unable to apply transition "approved" for workflow "group_request". " in the console command bin/console user-group:group-request when approving an item

Comment (ThomasTr) [reply config/packages/framework.yaml]: ok, have to revert also my change in GroupRequestCommand.php then it should work

**Comment by xabbuh** [comment]
Thanks @ThomasTr :+1:

**Pull request merged**